app/views/docs/email-and-sms-templates.phtml

```diff
@@ -0,0 +1,128 @@
+<p>Appwrite uses emails messages to perform user invite, user verification, login, and reset password. Emails messages can be customized to fit your app's design and voice.</p>
```

Comment:
The page is mostly focused on emails, while we also support SMS templates and maybe other in the future, we need to make the docs more balanced. I would also mention you can customize them per project and for multiple languages at the top of the page linking to more information on how to manage localization in Appwrite.

Reply:
I might even add a table of content for quick links to emails vs sms.

Reply:
Localization might be worth its own section with a code example.

Reply:
😆 Some of these sections are new, for a while I thought SMS delivery wasn't going to be added yet.